src/test/ui/consts/const-eval/issue-70723.rs

```diff
@@ -0,0 +1,5 @@
+#![feature(const_loop)]
+
+static _X: () = loop {}; //~ ERROR could not evaluate static initializer
```

Comment:
how long does this test take? If more than a second, please lower the step limit

Reply:
It takes not even 200ms here:
```
$ time rustc +rustc.2-s1 src/test/ui/consts/const-eval/issue-70723.rs 
error[E0080]: could not evaluate static initializer
 --> src/test/ui/consts/const-eval/issue-70723.rs:3:17
  |
3 | static _X: () = loop {}; //~ ERROR could not evaluate static initializer
  |                 ^^^^^^^ exceeded interpreter step limit (see `#[const_eval_limit]`)

error: aborting due to previous error

For more information about this error, try `rustc --explain E0080`.

real    0m0,125s
user    0m0,064s
sys     0m0,029s
```